presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -74,7 +74,17 @@ public void testInsert()
     @Override
     public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
     {
-        // TODO Support these test once kudu connector can create tables with default partitions
```

Comment:
`testDataMappingSmokeTest` is a generic test that covers a few different cases.

keep this TODO here and add your test as a new test method, eg `testPredicatePushdown`